Common/src/Storage/IObjectStorage.cs

```diff
@@ -92,21 +92,6 @@ public static IObjectStorage CreateResourcesStorage(this IObjectStorageFactory f
 /// </summary>
 public interface IObjectStorage
 {
-  /// <summary>
-  ///   Add the given data in the storage at the given key
-  ///   Update data if it already exists
-  /// </summary>
-  /// <param name="key">Key representing the object</param>
-  /// <param name="valueChunks">Chunks of data that will be stored in the Object Storage</param>
-  /// <param name="cancellationToken">Token used to cancel the execution of the method</param>
-  /// <returns>
-  ///   Task representing the asynchronous execution of the method
-  /// </returns>
-  /// <exception cref="ArmoniKException">there is 0 chunk</exception>
-  Task AddOrUpdateAsync(string                   key,
```

Comment:
Why did you remove this method from the interface?

Reply:
seen with you